internal/commands/secrets/create_inputs.go

```diff
@@ -0,0 +1,29 @@
+package secrets
+
+import (
+	"github.com/10gen/realm-cli/internal/app"
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/AlecAivazis/survey/v2"
+)
+
+type createInputs struct {
+	app.ProjectInputs
+	Name  string
+	Value string
+}
+
+func (i *createInputs) Resolve(profile *cli.Profile, ui terminal.UI) error {
+	if i.Name == "" {
+		if err := ui.AskOne(&i.Name, &survey.Input{Message: "Secret Name"}); err != nil {
+			return err
+		}
+	}
+	if i.Value == "" {
+		if err := ui.AskOne(&i.Value, &survey.Input{Message: "Secret Value"}); err != nil {
```

Comment:
ok this is the right idea (if input is blank, ask the question), but we can handle this a little differently with the way out `inputs` work.  check out the way `login/inputs.go` works, where we collect a slice of `*survey.Question`s and then prompt with:

```go
if err := ui.Ask(i, questions...); err != nil {
  return err
}
```

They key is that `survey.Question` takes a `Name` field which maps to field names found on `i`, binding the answers per question to each field